Containers/Deploy-.NET-Core-app-in-Windows-Containers-to-ECS/Readme.md

```diff
@@ -116,46 +116,50 @@ In this module we will configure a cluster for hosting our application using Ama
 
 ## Module 3: Create an ASP.NET Core Application
 
-In this module we will create a default ASP.NET Core MVC web application using the Microsoft-supplied project template.
+In this module we will create a default ASP.NET Core Web App using the Microsoft-supplied project template.
 
 - Time to Complete---10 mins
 
 ### Step 1: Create Application in Visual Studio for Windows
 
-If your development environment is Visual Studio 2017 or 2019 on Windows, you can create an ASP.NET Core application as follows:
+If your development environment is Visual Studio 2019 on Windows, you can create an ASP.NET Core application as follows:
```

Comment:
Add a note that Visual Studio 2017 users should follow their version of the New Project wizard (otherwise it reads like this can't be done in VS2017)

Reply:
Done.